frontend/src/components/AdminEditModal/AdminEditModal.css

```diff
@@ -0,0 +1,45 @@
+.edit-user-modal {
+  text-align: center;
+  margin-top: 125px;
+}
+
+.edit-user-modal .modal-content {
+  background: #f2f2f2;
+  border: none;
+}
+
+.edit-user-modal .modal-header {
+  padding-left: 30%;

```

Comment:
Most of the padding I've seen so far is using `px` rather than `%`, is there a specific reason this one works better as a percent? Have you checked at different widths if it looks good?

Reply:
Good point. Should probably keep it consistent.